### PR TITLE
WellFormed Broken

### PR DESCRIPTION
**Pull request by chut2870**
its broken now

### CODE REVIEW

**Comment by twang** [comment]
Thanks @chut2870!

@Shivmo:  if you review this late, please reopen the pull request

File: Wellformed.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+public class WellFormed
+    public static void main (String []args) {
+        System.out.println( "A System.out.println statement");
```
Comment (Shivmo) [inline Wellformed.java]: If you look on Practice It, it wants you to print 2 full paragraphs, not just the second paragraph stating with "A System..."

File: Wellformed.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+public class WellFormed
+    public static void main (String []args) {
+        System.out.println( "A System.out.println statement"); 
+        System.out.println( "has ( and ) and usually a");
+        System.out.println("String that starts and ends");
+        System.out.println("with a \" character.");
+         System.out.println("But we type \" instead!)");
```
Comment (Shivmo) [inline Wellformed.java]: Forgetting an open parenthesis "("

Comment (Shivmo) [inline Wellformed.java]: We want to print a slash as well as a quotation mark, but typing only "\"" will print only a quotation mark. We need to enter "\\\"" as we want to print both a back slash as well as a quotation mark.

File: Wellformed.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+                           }
+                           }
```
Comment (Shivmo) [inline Wellformed.java]: Indentation really helps me organize my code, and I think it could help you too! It really helps me with scope as well!

**Comment by twang** [comment]
Thanks @Shivmo!

**Pull request closed without merging**